.appveyor.yml

```diff
@@ -1,13 +1,12 @@
 environment:
   matrix:
-  - nodejs_version: "6.9"
+    - nodejs_version: "6"
```

Comment:
By setting `nodejs_version` to `6` it will takes the latest version of 6. Then, it will be resolved to 6.10.2 satisfying the requirement of 6.9 minimum. That's not what we want? Or you want to be sure that it was tested specifically with 6.9?